extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -373,4 +380,12 @@ public Duration getConnectionTimeout() {
     public void setConnectionTimeout(Duration connectionTimeout) {
         this.connectionTimeout = connectionTimeout;
     }
+
+    public OptionalInt getWebClientMaxPoolSize() {
+        return webClientMaxPoolSize;
+    }
+
+    public void setWebClientMaxPoolSize(OptionalInt webClientMaxPoolSize) {
```

Comment:
@glefloch the other pattern which we follow is in setters for the optional values it is a non opitonal parameter but internally we convert it to Optional - this looks strange - but the idea is the users know they set a concrete values, while the internal code deals with the opitonal value